src/index.html

```diff
@@ -258,12 +258,17 @@ <h4 class="card-title">
                 <button
                   class="btn btn-primary btn-lg btn-block mb-3"
                   id="getEncryptionKeyButton"
-                  disabled
```

Comment:
Why was this removed? The intention was to have all of the buttons that rely upon an account being connected to start as disabled until we know whether something is connected or not.

Reply:
The same goes for the "decrypt" button.